src/MakersOfDenmark.Application/Queries/V1/SearchForMakerSpace.cs

```diff
@@ -12,39 +12,39 @@
 
 namespace MakersOfDenmark.Application.Queries.V1
 {
-    public class GetSelectionOfMakerSpaces : IRequest<GetSelectionOfMakerSpacesResponse>
+    public class SearchForMakerSpace : IRequest<SearchForMakerSpaceResponse>
     {
-        public GetSelectionOfMakerSpaces(string name)
+        public SearchForMakerSpace(string name)
         {
             Name = name;
         }
 
         public string Name { get; }
     }
     
-    public class GetSelectionOfMakerSpacesHandler : IRequestHandler<GetSelectionOfMakerSpaces, GetSelectionOfMakerSpacesResponse>
+    public class SearchForMakerSpaceHandler : IRequestHandler<SearchForMakerSpace, SearchForMakerSpaceResponse>
     {
         private readonly MODContext _context;
 
-        public GetSelectionOfMakerSpacesHandler(MODContext context)
+        public SearchForMakerSpaceHandler(MODContext context)
         {
             _context = context;
         }
 
-        public async Task<GetSelectionOfMakerSpacesResponse> Handle(GetSelectionOfMakerSpaces request, CancellationToken cancellationToken = default)
+        public async Task<SearchForMakerSpaceResponse> Handle(SearchForMakerSpace request, CancellationToken cancellationToken = default)
         {
             var makerSpace = await _context.MakerSpace.Include(x => x.Address).AsNoTracking().FirstOrDefaultAsync(x => x.Name == request.Name);
```

Comment:
Egentligt så burde det være en Where(x => x.Name.Contains(request.Name)).ToListAsync() for at det er en søgning.

.Contains bliver oversat til en LIKE i SQL.

Reply:
Det gør så at dit response skal indeholde en liste af makerspaces og her skal det være makerspaceviewmodels.

Reply:
Er egentlig helt enig i, at det bør være en liste, der returneres. Status ændret til WIP

Reply:
Svaret kan jo være List < MakerSpaceViewModel > eller lignende... Så behøvs du ikke en response-klasse

Reply:
+1 God ide jesp, det får jeg lige kigget på.